src/main/scala/ScalaModulePlugin.scala

```diff
@@ -11,9 +11,13 @@ import sbt.librarymanagement.ivy.IvyDependencyResolution
 import sbt.librarymanagement.{ UnresolvedWarningConfiguration, UpdateConfiguration }
 
 object ScalaModulePlugin extends AutoPlugin {
-  val repoName            = settingKey[String]("The name of the repository under github.com/scala/.")
-  val mimaPreviousVersion = settingKey[Option[String]]("The version of this module to compare against when running MiMa.")
-  val scalaVersionsByJvm  = settingKey[Map[Int, List[(String, Boolean)]]]("For a Java major version (6, 8, 9), a list of a Scala version and a flag indicating whether to use this combination for publishing.")
+  object autoImport {
+    val repoName = settingKey[String]("The name of the repository under github.com/scala/.")
```

Comment:
sbt Plugins Best Practice recommends that we use plugin-specific prefix for all keys to avoid conflict - https://www.scala-sbt.org/1.x/docs/Plugins-Best-Practices.html#Key+naming+convention%3A+Use+prefix

If we are making these keys available automatically via autoImport, we should use our own prefix.